lib/spack/spack/test/cmd/release_jobs.py

```diff
@@ -36,6 +39,11 @@ def test_specs_deps(tmpdir, config):
     validate(deps_object, specs_deps_schema)
 
 
+@pytest.mark.skipif(
+    sys.version_info[:2] < (2, 7),
+    reason="For some reason in Python2.6 we get a utf-32 string "
+           "that can't be parsed"
+)
```

Comment:
Done

Reply:
I removed this change, because this test leaves fixtures etc. in a bad state when failing. See https://travis-ci.org/spack/spack/jobs/542632380 We probably need to refactor it. I'll open an issue so that we don't forget.